pages/_app.js

```diff
@@ -30,17 +32,29 @@ export default function App({ Component, pageProps }) {
     }
   }
 
+function handleToggleFavourites({ id }) {
```

Comment:
I would expect that just the id is passed, here you destructure an object that contains the id key.